src/test/resources/org/magicdgs/readtools/tools/distmap/DownloadDistmapResult/pg/equal/parts-00000-00003.pg.sam

```diff
@@ -0,0 +1,2027 @@
+@SQ	SN:211000022279174	LN:1195	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279175	LN:1144	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279177	LN:6076	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279178	LN:2438	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279179	LN:3073	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279180	LN:1011	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279181	LN:1148	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279182	LN:1277	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279183	LN:1231	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279185	LN:1210	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279186	LN:3102	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279188	LN:3347	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279189	LN:11958	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279190	LN:2575	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279191	LN:2549	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279192	LN:1843	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279193	LN:2462	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279195	LN:1022	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279197	LN:2369	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279198	LN:2030	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279199	LN:1872	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279200	LN:1008	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279203	LN:1116	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279204	LN:1458	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279207	LN:1484	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279208	LN:1761	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279211	LN:1056	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279212	LN:1126	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279214	LN:2555	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279215	LN:1012	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279217	LN:1966	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279218	LN:1246	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279221	LN:1119	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279222	LN:1136	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279224	LN:3173	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279227	LN:1098	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279228	LN:1933	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279229	LN:3490	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:211000022279230	LN:1775	AS:dmel6.03_wMel_lbrev_apast_lplant_dcv.nix
+@SQ	SN:21
```

Comment:
Look at this @robmaz - with your custom script, because the input files are the same (and not randomly generated), ReadTools is already merging properly the headers (as Picard would do).